backend/model/container_profile.rb

```diff
@@ -14,7 +14,7 @@ class ContainerProfile < Sequel::Model(:container_profile)
 
 
   def display_string
-    name
+    "#{name} [#{depth}d, #{height}h, #{width}w #{dimension_units}] extent measured by #{extent_dimension}"
```

Comment:
Hmm, not sure. The straight values are not capitalized which is how Maureen specified it.